### ISSUE TITLE
static.rust-lang.org/dist/ does not work anymore

### ISSUE THREAD

**Issue by nagisa** [rust-lang/rust-www#446]
It offers to download 0-sized file instead of providing a listing.

**Comment by nagisa** [comment]
cc @brson

**Comment by steveklabnik** [comment]
IIRC this was a decision, you have to type index.html yourself now, thanks to limitations on CloudFront. Or S3. Or something.

**Comment by nagisa** [comment]
It used to work before and [this](http://stackoverflow.com/questions/15309113/amazon-cloudfront-doesnt-respect-my-s3-website-buckets-index-html-rules) suggests you can in fact make this work with cloudfront too.

**Comment by alexcrichton** [comment]
Yes this was changed to ensure SSL is used all the way to S3. The previous configuration, and the one mentioned there I believe, involves cloudfront talking to S3 over HTTP, not HTTPS. Once we discovered that we remedied it quickly.

Unfortunately it looks like there's no way right now to have both `foo/` and `foo/index.html` work while having SSL all the way to S3, so there's not much we can do about this, so closing.
